### DISCUSSION TITLE
Project structure

### DISCUSSION THREAD

**Discussion by josephktcheung** [pointfreeco/isowords#138] [Q&A] [answered]
Hi @stephencelis @mbrandonw I'm trying to replicate the project structure, however I'm not able to hide the app directory inside the package like what isowords does. Is there anything more I add to set?

<img width="271" alt="Screenshot 2021-10-11 at 7 12 05 PM" src="https://user-images.githubusercontent.com/4270232/136781285-2498b913-73fe-4798-8495-13f083347692.png">

btw this is the sample repo I refer to https://github.com/bscothern/ExampleAppForSwiftPackage (related SO question https://stackoverflow.com/questions/59800045/how-can-i-add-a-demo-app-target-to-a-swift-package)

**Comment by mbrandonw** [comment, marked as answer]
Hi @josephktcheung, good question. If you add a `Package.swift` in a directory with the minimal amount of information to be valid, then Xcode will not display it in the package. For example, in our `App` directory we have this:

https://github.com/pointfreeco/isowords/blob/main/App/Package.swift

**Reply by josephktcheung** [reply]
@mbrandonw got it working now, thanks for your help!
